File: app/src/main/java/com/example/meeera/browserapp/Model/HistoryModel.java
package com.example.meeera.browserapp.Model;

import io.realm.RealmObject;

/**
 * Created by meeera on 6/10/17.
 */

public class HistoryModel extends RealmObject {

    public String history;

    public String getHistory() {
        return history;
    }

    public void setHistory(String history) {
        this.history = history;
    }
}

File: app/src/main/java/com/example/meeera/browserapp/Model/HistoryModel.kt
package com.example.meeera.browserapp.Model

import io.realm.RealmObject

/**
 * Created by meeera on 4/10/17.
 */
class HistoryModel : RealmObject() {
    var history : String ?= null

    fun getHistoryData(): String {
        return history.toString()
    }

    fun setHistoryData(data: String) {
        history = data
    }

}